app/views/users/index.html.erb

```diff
@@ -1,67 +1,17 @@
-<div class="col-lg-6 mx-auto">
-  <div class="card mb-3" style="max-width: 540px;">
-    <div class="row g-0">
-      <div class="col-md-4">
-        <img src="https://unsplash.com/photos/F_-0BxGuVvo" class="img-fluid rounded-start" alt="...">
-      </div>
-      <div class="col-md-8">
-        <div class="card-body">
-          <h5 class="card-title">Tom</h5>
-          <p class="card-text"><small class="text-muted">Number of posts: 1</small></p>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div class="card mb-3" style="max-width: 540px;">
-    <div class="row g-0">
-      <div class="col-md-4">
-        <img src="..." class="img-fluid rounded-start" alt="...">
-      </div>
-      <div class="col-md-8">
-        <div class="card-body">
-          <h5 class="card-title">User name</h5>
-          <p class="card-text"><small class="text-muted">Number of posts: x</small></p>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div class="card mb-3" style="max-width: 540px;">
-    <div class="row g-0">
-      <div class="col-md-4">
-        <img src="..." class="img-fluid rounded-start" alt="...">
-      </div>
-      <div class="col-md-8">
-        <div class="card-body">
-          <h5 class="card-title">User name</h5>
-          <p class="card-text"><small class="text-muted">Number of posts: x</small></p>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div class="card mb-3" style="max-width: 540px;">
-    <div class="row g-0">
-      <div class="col-md-4">
-        <img src="..." class="img-fluid rounded-start" alt="...">
-      </div>
-      <div class="col-md-8">
-        <div class="card-body">
-          <h5 class="card-title">User name</h5>
-          <p class="card-text"><small class="text-muted">Number of posts: x</small></p>
+<% @users.each do |user| %>
+  <div class="col-lg-6 mx-auto">
+    <div class="card mb-3" style="max-width: 540px;">
+      <div class="row g-0">
+        <div class="col-md-4">
+          <img src="<%= user.photo%>" class="img-fluid rounded-start" alt="...">
         </div>
-      </div>
-    </div>
-  </div>
-  <div class="card mb-3" style="max-width: 540px;">
-    <div class="row g-0">
-      <div class="col-md-4">
-        <img src="..." class="img-fluid rounded-start" alt="...">
-      </div>
-      <div class="col-md-8">
-        <div class="card-body">
-          <h5 class="card-title">User name</h5>
-          <p class="card-text"><small class="text-muted">Number of posts: x</small></p>
+        <div class="col-md-8">
+          <div class="card-body">
+            <h5 class="card-title"><%= link_to  user.name, user %></h5>
```

Comment:
- [ ] On starting the app and going to the users page, I got this error. Kindly fix it.
![Capture](https://user-images.githubusercontent.com/85095702/189017459-7d765372-f16e-4ee4-9fd8-31f94c3a7a0b.PNG)

- [ ] Also, it will be nice if your root page is the users page instead of the rails default page.

Reply:
I changed the root to users#index, but I don't know why are you having this issue 👆🏽 I recorded a video where you can see the blog app in function:

https://www.loom.com/share/75766e10bb59457f9a8d4a27fcfdabcb